### PR TITLE
Update community-meetups.json

### PR DESCRIPTION
**Pull request by duckdegen**
Adding CryptoCanal events for Amsterdam, Rotterdam and Zurich

## Description

Added three cities with CryptoCanal meetups: Amsterdam, Rotterdam and Zurich

### CODE REVIEW

File: src/data/community-meetups.json [hunk 1]
```diff
@@ -1,4 +1,22 @@
...
+    "location": "Amsterdam",
+    "link": "https://www.cryptocanal.org/agenda"
+  },
+  {
+    "title": "CryptoCanal",
+    "emoji": ":ch:",
+    "location": "Zurich",
+    "link": "https://www.cryptocanal.org/agenda"
+  },
+  {
```
Comment (wslyvh) [inline src/data/community-meetups.json:14]: the Netherlands is too small to add the same meetup for multiple cities imo 
Maybe just update location to 'the Netherlands' instead?

Comment (duckdegen) [reply src/data/community-meetups.json:14]: Sure, can do!

Comment (wslyvh) [reply src/data/community-meetups.json:14]: I think its fine to add CryptoCanal as 2 location for the Netherlands and Zurich, Swiss.
That makes sense. Someone browsing through the list is looking for meetups in his city/country.

But no need to differentiate between Amsterdam/Rotterdam imo.

**Review by corwintines — APPROVED** [review]
Thanks @duckdegen!

**Pull request merged**